src/github.ts

```diff
@@ -461,17 +462,24 @@ export class GitHub {
   // the branch we're configured for.
   async latestTag(
     prefix?: string,
-    preRelease = false
+    preRelease = false,
+    // Allow a branch prefix to differ from a tag prefix. This was required
+    // for google-cloud-go, which uses the tag prefix library/vx.y.z.
+    // this is a requirement in the go community.
+    branchPrefix?: string
```

Comment:
Is this still needed with the new separator method?

Reply:
Unfortunately yes, because the very first release of each library will hit `latestTagFallback`, as we've never created a branch in the `release-[library]-version` format before.

Reply:
ah, gotcha.